trantor/utils/Utilities.h

```diff
@@ -0,0 +1,117 @@
+/**
+ *
+ *  @file Utilities.h
+ *  @author An Tao
+ *
+ *  Copyright 2018, An Tao.  All rights reserved.
+ *  https://github.com/an-tao/drogon
+ *  Use of this source code is governed by a MIT license
+ *  that can be found in the License file.
+ *
+ *  Drogon
+ *
+ */
+
+#pragma once
+
+#include <trantor/exports.h>
+#include <string>
+
+namespace trantor
+{
+namespace utils
+{
+/**
+ * @brief Convert a wide string to a UTF-8.
+ * @details UCS2 on Windows, UTF-32 on Linux & Mac
+ *
+ * @param str String to convert
+ *
+ * @return converted string.
+ */
+TRANTOR_EXPORT std::string to_utf8(const std::wstring &wstr);
+/**
+ * @brief Convert a UTF-8 string to a wide string.
+ * @details UCS2 on Windows, UTF-32 on Linux & Mac
+ *
+ * @param str String to convert
+ *
+ * @return converted string.
+ */
+TRANTOR_EXPORT std::wstring from_utf8(const std::string &str);
+
+/**
+ * @details Convert a wide string path with arbitrary directory separators
+ * to a UTF-8 portable path for use with trantor.
+ *

```

Comment:
Although I'm a supporter of snake_case. Please use Camel case since that's the coding style.

Reply:
Oh yes, you're right.
I didn't take care when I initially made them in drogon.
And when I moved them to trantor, I didn't thought about renaming them.
I'll quickly do that.